src/Morphir/IR/Source.elm

```diff
@@ -73,6 +87,28 @@ type alias DataSourceName =
     Name
 
 
+{-| Represents a Literal type.
+The fields of a Literal are:
```

Comment:
I don't think these are fields, they are constructors, right?

Reply:
Yes, you're right. They are constructors